app/src/main/java/com/pintraveler/ptkit/MainActivity.kt

```diff
@@ -0,0 +1,11 @@
+package com.pintraveler.ptkit
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class MainActivity : AppCompatActivity() {
```

Comment:
ille lazim mi bir activity tanimlamamiz?

Reply:
internette birçok örnekte bu şekilde yapılıyordu. sanırım daha öncesinde böyle olmadığı için jitpack de hata alıyorduk